develop/TOOLS/DIS/Concepts/Overall_concept_of_the_DataMiner_Integration_Studio.md

```diff
@@ -16,12 +16,23 @@ After installation of the DataMiner Integration Studio extension, Microsoft Visu
 - A function editor, which allows you to manage the functions defined in a function XML file and design function pages using simple drag-and-drop operations.
 - A table editor, which allows you to design parameter tables using simple drag-and-drop operations.
 
-> [!NOTE]
-> DIS will set the language version of the C# projects of 4.0, 6.0 or 7.3 depending on the DMA version specified in the *Protocol.Compliancies.MinimumRequired* tag of the protocol XML file:
-> - If the DMA version is lower than “9.6.11” (or if no *Protocol.Compliancies.MinimumRequired* tag could be found), the language version of the C# projects will be set to 4.0.
-> - If the DMA version is equal to or higher than “9.6.11”, the language version of the C# projects will be set to one of the following versions:
->     - C# 6.0 (when using Visual Studio 2015)
->     - C# 7.3 (when using Visual Studio 2017 or above)
+### Language version of C# projects
+
+#### [Up to DIS v2.37](#tab/tabid-1)
+
+DIS will set the language version of the C# projects of 4.0, 6.0 or 7.3 depending on the DMA version specified in the *Protocol.Compliancies.MinimumRequired* tag of the protocol XML file:
+
+- If the DMA version is lower than 9.6.11 (or if no *Protocol.Compliancies.MinimumRequired* tag could be found), the language version of the C# projects will be set to 4.0.
+- If the DMA version is equal to or higher than 9.6.11, the language version of the C# projects will be set to one of the following versions:
+
+    - C# 6.0 (when using Visual Studio 2015)
+    - C# 7.3 (when using Visual Studio 2017 or above)
+
+#### [As from DIS v2.38](#tab/tabid-2)
```

Comment:
Did not compile the help to verify but from reading this, it feels to me like "As from DIS v2.38" will be the second tab? (just ignore this comment if I'm miss-interpreting the code here.)
While it should be the first tab and the one active by default.
(we want to, by default, show the current & up to date info. The other tabs are there to have all historical info if needed but should typically not be used that much)

Reply:
indeed. well spotted. as from 2.38 will now be first tab.